products/sle/main.pm

```diff
@@ -1140,7 +1140,7 @@ elsif (is_kgraft) {
 }
 elsif (get_var("EXTRATEST")) {
     prepare_target();
-    load_extra_test();
+    load_extra_test() unless check_var('HPC', 'master');
```

Comment:
that was meant as a temporary hack to test the module more easily for you locally. Please discuss with @Soulofdestiny how to trigger the HPC tests in all cases correctly living under the constraints of not having our cool whiteboard ideas regarding better test plans already implemented ;-)